src/Core/Manifest/VersionProvider.php

```diff
@@ -145,14 +145,14 @@ private function setCacheValue(string $key, string $value): void
      *   cwp/cwp-core => ['CWP', '4.4.4']
      * ]
      *
-     * @param array $modules
-     * @return array
+     * @param array<string,mixed> $modules
+     * @return array<string,mixed>
```

Comment:
Where does this `mixed` come from?

Reply:
well, that's my default take when i'm not exactly sure what this will do, i know there a string keys, but values not sure so, so i go for string,mixed. which is better than just "array" anyway
it's always possible to improve this further if we make it clear what this should contain

Reply:
If we don't know for sure it's mixed, we're better off not saying anything about the type. Please refer this change.

Reply:
it should be      
* @param array<string,array<int,string>> $modules
* @return array<string,array<int,string>>